arch/arm/boot/dts/overlays/Makefile

```diff
@@ -309,7 +309,8 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	waveshare-can-fd-hat-mode-a.dtbo \
 	waveshare-can-fd-hat-mode-b.dtbo \
 	wittypi.dtbo \
-	wm8960-soundcard.dtbo
+	wm8960-soundcard.dtbo \
+	panel-dsi-generic.dtbo
```

Comment:
These seem to be sorted alphabetically and putting this at the end of the list breaks the sort.

Reply:
oh true